### ISSUE TITLE
TorBrowserBundle: don't modify install destination "/Applications"

### ISSUE THREAD

**Issue by workflowsguy** [autopkg/homebysix-recipes#167]
Thanks for providing this recipe.
I would like to use it, but am having a problem with it since you move the installed software from "/Applications" to "~/Applications"  in a post-install script.
I think such modifications should go into a custom recipe.

I would like to get the original behaviour back but I do not know how to undo the effects of this post-install script in a recipe override.

**Comment by homebysix** [comment]
Normally Mac applications would be stored in the /Applications folder, but I made an exception for Tor Browser. [At last check](https://trac.torproject.org/projects/tor/ticket/13252), the Tor Browser stored some data in the application bundle itself, and therefore storing the application bundle in a shared location was not desirable.

I haven't been following along closely, but it looks like the linked issue was closed. Maybe that's a good sign that it's safe to store the app in the standard /Applications path now. Open to PRs that make that change.
